### PR TITLE
MC-13973 Predefined edge rules api docs

### PR DESCRIPTION
**Pull request by y0geshdev**
#### Changes made

-  Added api docs for list and edit predefined edge rules

### CODE REVIEW

File: source/includes/stackpath/_predefined_edge_rules.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/predefinededgerules/<a href="#stackpath-site">:siteId</a></code>
+
+Retrieve the configuration of all predefined edge rules in a given [environment](#administration-environments) within a site.
+
+Attributes | &nbsp;
+------- | -----------
+`allowEmptyReferrer`<br/>*boolean* | Whether or not empty referrer is allowed.
+`forceWwwEnabled`<br/>*boolean* | Whether or not redirecting every request to a www subdomain is enabled.
+`id`<br/>*UUID* | A predefined egde rules identifier which is same as siteId.
```
Comment (razine-bensari) [inline source/includes/stackpath/_predefined_edge_rules.md]: I fins this sentence ambiguous, is it the same as the SiteId or the same type (UUID).

```suggestion
`id`<br/>*UUID* | A predefined egde rules identifier which is the same as a siteId.
```
if the latter then the suggestion above is what I would have used

**Review by razine-bensari — APPROVED** [review]
lgtm, I added a comment that is worth addressing I think

File: source/includes/stackpath/_predefined_edge_rules.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+### Predefined Edge Rules
+
+The predefined edge rules let you configure how StackPath responds to requests to your website. These predefined rules only work with domains that resolve to StackPath.
```
Comment (m-orfanos) [inline source/includes/stackpath/_predefined_edge_rules.md]: Should we rename `edge rules` to `EdgeRules`?

**Review by m-orfanos — CHANGES_REQUESTED** [review]
just a small question

File: source/includes/stackpath/_predefined_edge_rules.md (hunk 1 above)
Comment (y0geshdev) [reply source/includes/stackpath/_predefined_edge_rules.md]: It's former. The predefinedEdgeRuleId is actually the siteId as there is no entity in SP// which maps to predefined edge rule, rather it's a combination of some settings.
How about - 
`This ID is same as the siteId to which the predefined rules belong.`

File: source/includes/stackpath/_predefined_edge_rules.md (hunk 2 above)
Comment (y0geshchaudhary) [reply source/includes/stackpath/_predefined_edge_rules.md]: 852e6a9

Comment (y0geshdev) [reply source/includes/stackpath/_predefined_edge_rules.md]: Updated different variations of `edge rules` by `EdgeRules` based on SP// docs and usage.
~~852e6a9~~

ddf480c

File: source/includes/stackpath/_predefined_edge_rules.md (hunk 1 above)
Comment (y0geshdev) [reply source/includes/stackpath/_predefined_edge_rules.md]: ~~852e6a9~~

ddf480c

**Review by m-orfanos — APPROVED** [review]
Thanks for the changes! :)

**Pull request merged**